examples/pytorch/TranslationFairseq/README.md

```diff
@@ -0,0 +1,62 @@
+## Fairseq en to de translation example with MLflow
+
+This example illustrates the training and testing process of English to German translation model using [fairseq](https://ai.facebook.com/tools/fairseq/) and its integration with MLflow.
+During the process, all the parameters, metrics, summary and the model were dumped into mlflow. 
+
+### Package Requirements
+
+Install the required packages mentioned in the following fairseq readme link
+
+https://pytorch.org/hub/pytorch_fairseq_translation/
+
+In addition to these, following pip packages are used
+
+1. prettytable
+2. ConfigParser
+3. mlflow
+4. pyximport
+
+### Clone the repository
+
+Follow the `Requirements and Installation` section from the following link to clone the repository
+
+https://github.com/pytorch/fairseq
+
+### Replace training File
+
+Once repository is cloned, replace `train.py` with `fairseq_cli/train.py` file
+
+### Setting MLflow Configuration
+
+Set the MLflow tracking Uri and experiment name in mflow.cfg file and copy it to the package root directory.
+
+### Building fairseq from source
+
+Run the following command to build the fairseq package
+
+`python setup.py build_ext --inplace`
+`python setup.py install`
+
+### Preprocessing
+
+Run the command from section - `WMT'14 English to German (Convolutional)` to preprocess
+
```

Comment:
@ankan94  in the referenced readme there is sequence of commands, which clarifying which commands needs to be run would help the user to better follow.